railties/lib/rails/generators/app_base.rb

```diff
@@ -623,7 +623,7 @@ def css_gemfile_entry
       end
 
       def cable_gemfile_entry
-        return if options[:skip_action_cable]
+        return if options[:skip_action_cable] || !options[:skip_solid]
 
         comment = "Use Redis adapter to run Action Cable in production"
         GemfileEntry.new("redis", ">= 4.0.1", comment, {}, true)
```

Comment:
Feel free to ignore this cosmetic suggestion. In my brain I think "if action cable is enabled and solid is disabled, then mention Redis in the Gemfile". This would translate to something like the following:

```suggestion
        if !options[:skip_action_cable] && options[:skip_solid]
          comment = "Use Redis adapter to run Action Cable in production"
          GemfileEntry.new("redis", ">= 4.0.1", comment, {}, true)
        end
```